charts/internal/control-plane/templates/auditlog-webhook-deployment.yaml

```diff
@@ -0,0 +1,124 @@
+{{- if .Values.auditlogWebhook.enabled }}
+---
+# Probably shouldnt need this when all is in place:
+# apiVersion: v1
+# data:
+#   ca.crt: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUVDVENDQXZFQ0ZDdkk3cjE2SWkwNzJ0Ukp5djZIbFNQZFo5N2NNQTBHQ1NxR1NJYjNEUUVCRFFVQU1JSEEKTVFzd0NRWURWUVFHRXdKRVJURVFNQTRHQTFVRUNBd0hRbUYyWVhKcFlURVFNQTRHQTFVRUJ3d0hRbUYyWVhKcApZVEVPTUF3R0ExVUVFUXdGT0RVMU5EQXhJVEFmQmdOVkJBa01HRkpwWTJoaGNtUXRVbVZwZEhwdVpYSXRRV3hzClpXVWdPREVPTUF3R0ExVUVDZ3dGUmtrdFZGTXhGVEFUQmdOVkJBc01ER05zYjNWa0xXNWhkR2wyWlRFUk1BOEcKQTFVRUF3d0labWt0ZEhNdWFXOHhJREFlQmdrcWhraUc5dzBCQ1FFV0VXNXZMWEpsY0d4NVFHWnBMWFJ6TG1sdgpNQjRYRFRJd01EUXlNakUzTkRVME5Gb1hEVEl3TURVeU1qRTNORFUwTkZvd2djQXhDekFKQmdOVkJBWVRBa1JGCk1SQXdEZ1lEVlFRSURBZENZWFpoY21saE1SQXdEZ1lEVlFRSERBZENZWFpoY21saE1RNHdEQVlEVlFRUkRBVTQKTlRVME1ERWhNQjhHQTFVRUNRd1lVbWxqYUdGeVpDMVNaV2wwZW01bGNpMUJiR3hsWlNBNE1RNHdEQVlEVlFRSwpEQVZHU1MxVVV6RVZNQk1HQTFVRUN3d01ZMnh2ZFdRdGJtRjBhWFpsTVJFd0R3WURWUVFEREFobWFTMTBjeTVwCmJ6RWdNQjRHQ1NxR1NJYjNEUUVKQVJZUmJtOHRjbVZ3YkhsQVpta3RkSE11YVc4d2dnRWlNQTBHQ1NxR1NJYjMKRFFFQkFRVUFBNElCRHdBd2dnRUtBb0lCQVFEamtQeDQvQ3NNZTY5eElUOWlnald1bWwvQzlTalp3TXBKUXVmNwpzMXRZT08vVUUyQ1Uyb1JMN0xtSS9Cak12NHY4dmt1TytLZ1V2OXhPcGc1aWxJclNzZjFtTjF0blptd3Q2UnV4CitSZWN0ZFhHNXJlbTFjSUdsNEh0MEhsNTl2eVIyMVV2L0dORElacnNEVm5CREV4OWpQTFVNd0dPeW9rOG1pUVYKYld2Umx1V2dJeTFlR3g0OGcwRDhqRVBJMWttNGVaQzNWRGJ0Rnc1bVRzVDJEOXVja2tHbU1FcXNLdTdzV1pQTQpZdEF5a1ZMTEJEOTg5cXlzVjJrYUNmNEptV1RBMVVpT0RQR0FDL0Q3UDJVM1EvalB6dDFERlpUbHhzR2FDZ0JiClA2NmtSNEowZ0kyZmc3N1ZGeCtjQVJRWDdsUnY4VGJadENESnJLNFk1MmdIUnAvQkFnTUJBQUV3RFFZSktvWkkKaHZjTkFRRU5CUUFEZ2dFQkFHQUJ2SUdQTjgwYXBIYmdYQ2lrRGFvOWQ0bTE0M3ZRbkhtNHZkellGc2o4bjR0dgp4SEtPSWwreTdvdlRNVEJmdGJCUWNRUDR5VUVOS05iSG4rZFBOREZyL3ltZTUvQUdMM2RROXVoMXJtREpsSGk4CkJNbzhFYkVxMzMvSXFLVm55dEQzby9vYklzTjgvdFBSWGo4RUprWG5nMkpaNW5mRjZvaTRtK1RwTlJIQnNLRFAKT2xiUHRpYm9od2ZGT0hTOEtQMmhVWDRJQ2Q5MlI1YitsSnNYYTh4Q0VQdlhzaGF1RXpXcWIzS28xdUlIbitKagp3WFBTSDltLzBPWjIrK3pyVERaNzByVFBxd0hGTjB1eUNTK0t3b3hFbVR0UnVobnNGQXZsU2JFTmdQbEJXRmpECjFxYlJKSGxpWllYY2JTU3VHdU1sMGVheElBWFlZRWRGTDEzYW5GST0KLS0tLS1FTkQgQ0VSVElGSUNBVEUtLS0tLQo=
```

Comment:
You already indicated that we do not really need example certs here. Would be better to just remove them.